jmix-flowui/flowui/src/main/java/io/jmix/flowui/component/tabsheet/JmixTabSheet.java

```diff
@@ -427,4 +452,30 @@ public boolean isInitialSelection() {
             return this.initialSelection;
         }
     }
+
+    class LazyTabChangeListener implements ComponentEventListener<SelectedChangeEvent> {
```

Comment:
1. we don't use `package private` access
2. Since the `addLazyTab` method is `@Internal` and it's unlikely that someone will figure out what `ComponentLoader` should be passed, it'd rather get rid of this method and implement everything in the loader